docs/en/dev_setup/dev_env_mac.md

```diff
@@ -1,40 +1,27 @@
-# MacOS Development Environment
+# macOS Development Environment
 
 The following instructions set up a PX4 development environment for macOS.
 This environment can be used to build PX4 for:
 
 - Pixhawk and other NuttX-based hardware
-- [Gazebo Classic Simulation](../sim_gazebo_classic/index.md)
+- [Gazebo Simulation](../sim_gazebo_gz/index.md) (Gazebo Harmonic)
 
 :::tip
 This setup is supported by the PX4 dev team.
-To build other targets you will need to use a [different OS](../dev_setup/dev_env.md#supported-targets) (or an [unsupported development environment](../advanced/community_supported_dev_env.md)).
+To build other targets you will need to use a [different OS](../dev_setup/dev_env.md#supported-targets) or an [unsupported development environment](../advanced/community_supported_dev_env.md).
 :::
 
 ## Video Guide
 
 <lite-youtube videoid="tMbMGiMs1cQ" title="Setting up your PX4 development environment on macOS"/>
 
-## Base Setup
+## Development Environment Setup
 
-The "base" macOS setup installs the tools needed for building firmware, and includes the common tools that will be needed for installing/using the simulators.
+### Prerequisites
 
-### Environment Setup
-
-:::details Apple Silicon Macbook users!
-If you have an Apple M1, M2 etc. Macbook, make sure to run the terminal as x86 by setting up an x86 terminal:
-
-1. Locate the Terminal application within the Utilities folder (**Finder > Go menu > Utilities**)
-2. Select _Terminal.app_ and right-click on it, then choose **Duplicate**.
-3. Rename the duplicated Terminal app, e.g. to _x86 Terminal_
-4. Now select the renamed _x86 Terminal_ app and right-click and choose \*_Get Info_
-5. Check the box for **Open using Rosetta**, then close the window
-6. Run the _x86 Terminal_ as usual, which will fully support the current PX4 toolchain
-   :::
-
-First set up the environment
+1. Install Homebrew by following these [installation instructions](https://brew.sh).
 
-1. Enable more open files by appending the following line to the `~/.zshenv` file (creating it if necessary):
+2. Enable more open files by appending the following line to the `~/.zshenv` file (creating it if necessary):
```

Comment:
I don't use .zshenv. Is this the default thing that everyone uses for all shells on macos?